src/test/resources/db/user/delete-test-user.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM users_roles WHERE role_id = 1;
```

Comment:
if you want to test making changes in DB, it's not required to execute SQL scripts for restoring DB after tests, you can use `@Transactional @Rollback` on method annotations or test containers